.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    paths:
+      - 'charts/**'
+  pull_request:
+    paths:
+      - 'charts/**'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Run chart-testing (lint)
+        id: lint
+        uses: helm/chart-testing-action@v1.0.0
+        with:
+          command: lint
+          config: ct.yaml
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        with:
+          installLocalPathProvisioner: true
```

Comment:
What is the local Path provisioner here?

Reply:
It is related to the Kind cluster. I did not get a chance to look on that part.
cc @prateekpandey14 -- Can you help here? 
Anyways, I will take a look at this part.

Reply:
Its a rancher localpath provisioner for hostpath/localpv volume provisioning supported by kind based cluster to CI/CD test.
Setting `true` will be installing the provisioner as part of cluster create. Mostly its not required  in our case to test the charts, but if we want we can skip it.

Reply:
Okay. I will remove it.